internal/confmapprovider/discovery/bundle/bundle.d/receivers/rabbitmq.discovery.yaml.tmpl

```diff
@@ -0,0 +1,24 @@
+{{ receiver "rabbitmq" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)rabbitmq.*"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)rabbitmq.*" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)rabbitmq.*"
+  config:
+    default:
+      endpoint: '`endpoint`'
+      username: {{ defaultValue }}
+      password: {{ defaultValue }}
+      collection_interval: 10s
+  status:
+    metrics:
+      - status: successful
+        strict: rabbitmq.consumer.count
+        message: RabbitMQ receiver is working!
+    statements:
+      - status: failed
+        regexp: 'connect: network is unreachable'
+        message: The container cannot be reached by the Collector. Make sure they're in the same network.
+      - status: failed
+        regexp: 'connect: connection refused'
+        message: The container is refusing RabbitMQ server connections.
```

Comment:
nit Q.: can't this also happen for the`host_observer`?

Reply:
This is meant for all three discovery extensions ( i.e. host observer, k8s observer and docker observer ). What it is saying, that due to network issues Rabbitmq server is refusing any connections!!